core/sequencer/sequencer.go

```diff
@@ -58,6 +58,9 @@ var (
 	})
 )
 
+// MaxBatchSize limits the number of mutations that will be processed per epoch.
+var MaxBatchSize = 1000
```

Comment:
Batch?  Page?  Would be good to have consistent terminology with the `MutationStorage.Read` API itself.

Reply:
This isn't perfect, but I'm trying on `ReadPage` and `ReadBatch` for size. 
`ReadPage` lists mutations by page between a start and end sequence number.
`ReadBatch` lists mutations by batch between a start sequence number to the current sequence number.